apps/prairielearn/src/pages/partials/navbarPublic.ejs

```diff
@@ -0,0 +1,6 @@
+<li class="nav-item btn-group">
+    <!-- TODO make this link to public questions page for the course? -->
+    <a class="nav-link" role="button">
+        <%= course.short_name %>
+    </a>
+</li>
```

Comment:
For some reason I cannot figure out how to make the navbar fit the entire width of the window. Help with this would be appreciated. 

I don't understand how dropping in `navbarStudent`, `navbarInstitution`, etc. makes the navbar full width, all of them are just a bunch of list items.

The seemingly never used navbarPlain appears to have the same issue, it doesn't fill the whole window width.

Reply:
fixed